src/lancer.scss

```diff
@@ -156,7 +156,7 @@ button {
   font-family: var(--main-font-family);
 }
 
-.window-app.lancer .window-content {
+.window-app.lancer .window-content .form-notes {
```

Comment:
This does narrow the style to one of our custom apps, but there's a bunch of other apps that need to get targeted too. I started building a list, but then it occurred to me - why don't we just remove the `lancer` class from the settings apps? `src/module/apps/action-tracker-settings.ts`, `.../automation-settings.ts`, and `.../status-icon-config.ts` should be all, and I think that's more logical than specifying another class for every `.lancer` app to apply the styling.

Reply:
Ah, I see what you're saying. That does make more sense,; I'll make the adjustments.